### PR TITLE
docs(web-modeler): update the text and screenshot in the engine versi…

### PR DESCRIPTION
**Pull request by nithinssabu**
…on selection section

## What is the purpose of the change

_Briefly describe the change you are making, this helps the reviewer by providing an overview._
The design of the version selector component was updated, hence the documentation also need to be updated.

## Are there related marketing activities

_Are there any marketing activities related to the related feature or component? If so, please briefly describe them._
No.

## When should this change go live?

_Does this change need to be released before a certain date or milestone? If so, when?_
ASAP

## PR Checklist

- [x] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [x] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by akeller** [comment]
This PR only includes changes to `next`, not the current 8.1 docs. Is this intentional? Since web-modeler is a continuously releasing component it's likely you want to change both `next` and 8.1 docs.

File: docs/components/modeler/web-modeler/fix-problems-in-your-diagram.md [hunk 1]
```diff
@@ -12,7 +12,9 @@ Based on a set of lint rules, Web Modeler continuously validates implementation
 
 ### Engine version selection
 
-The version selector can be used to choose the Zeebe version the diagram is validated against. The version chosen should match the Zeebe version of the cluster the diagram is going to be deployed in to get the correct set of errors. The version selector also provides information about the number of clusters available for each Zeebe version within the current organization.
+The version selector at the top right in the problems panel can be used to choose the Zeebe version the diagram is validated against. The version chosen should match the Zeebe version of the cluster the diagram is going to be deployed in to get the correct set of errors (if you do not know the version, it is shown alongside the cluster name in the deployment dialog which can be opened by clicking the **Deploy diagram** button.)
```
Comment (christinaausley) [inline docs/components/modeler/web-modeler/fix-problems-in-your-diagram.md:15]: Do you mean properties panel instead of problems panel here?

**Review by christinaausley — CHANGES_REQUESTED** [review]
Technical review in latest commits to bold the UI element referenced. One comment for clarity.

File: docs/components/modeler/web-modeler/fix-problems-in-your-diagram.md (hunk 1 above)
Comment (christian-konrad) [reply docs/components/modeler/web-modeler/fix-problems-in-your-diagram.md:15]: Problems panel is right here :)

**Review by christian-konrad — APPROVED** [review]
Looks good, thank you!

**Comment by nithinssabu** [comment]
> This PR only includes changes to `next`, not the current 8.1 docs. Is this intentional? Since web-modeler is a continuously releasing component it's likely you want to change both `next` and 8.1 docs.

@akeller I believe this PR includes changes to both `next` and `8.1` docs. Am I missing something?

**Comment by akeller** [comment]
> @akeller I believe this PR includes changes to both `next` and `8.1` docs. Am I missing something?

@nithinssabu I see both sets of changes now, so this is good to go from that perspective. Thank you!

**Review by christinaausley — APPROVED** [review]

**Pull request merged**